app/assets/src/components/views/phylo_tree/PhyloTreeCreation.jsx

```diff
@@ -353,9 +355,30 @@ class PhyloTreeCreation extends React.Component {
     let nSamples =
       this.state.selectedProjectSamples.size +
       this.state.selectedOtherSamples.size;
-    return (
-      nSamples >= this.minNumberOfSamples && nSamples <= this.maxNumberOfSamples
+    return PhyloTreeChecks.isNumberOfSamplesValid(nSamples);
+  }
+
+  getNumberOfSamplesWithFewReads() {
```

Comment:
I would add a comment explaining that this is the number of samples with few reads in EITHER nt or nr (an important detail).

Reply:
Added!